Changelog.md

```diff
@@ -5,6 +5,9 @@ Language Features:
 
 Compiler Features:
  * Build system: Update the soljson.js build to emscripten 2.0.12 and boost 1.75.0.
+ * Command Line Interface: New option ``--model-checker-targets`` allows specifying which targets should be checked. The valid options are ``all``, ``constantCondition``,
```

Comment:
Should we consider `none` as an alternative to the pragma? Or selecting none for engine is still the better option? See #10166.

Reply:
In any case we should do that in a separate PR to keep these separate.

Reply:
Yea, I'd prefer to use `engine=none` for that.

Reply:
Is `constantCondition` the same as `unreachableCode`? Does it check for unreachable code?

Reply:
Yea, `constantCondition` means both unreachable code and trivial condition